### ISSUE TITLE
SIGSEV issue when loading node data

### ISSUE THREAD

**Issue by dannydeezy** [lncapital/torq#99]
Running torq locally on my machine w/ docker, which is fetching data from my cloud-hosted node.

On startup when I load the main webpage after logging in, it runs for a while then throws this error
```
Listening on port 8080
{"level":"info","time":"2022-09-26T16:35:57Z","message":"Subscribing to LND for node id: 1"}
[GIN] 2022/09/26 - 16:36:43 | 304 |     859.837µs |      172.19.0.1 | GET      "/"
[GIN] 2022/09/26 - 16:36:43 | 304 |     376.795µs |      172.19.0.1 | GET      "/static/css/main.575ac50b.css"
[GIN] 2022/09/26 - 16:36:43 | 200 |   67.138275ms |      172.19.0.1 | GET      "/static/js/main.3f206946.js"
[GIN] 2022/09/26 - 16:36:43 | 401 |     254.994µs |      172.19.0.1 | GET      "/api/settings"
[GIN] 2022/09/26 - 16:36:43 | 401 |     568.361µs |      172.19.0.1 | GET      "/api/flow?from=2022-09-20&to=2022-09-27&chan_id=1"
[GIN] 2022/09/26 - 16:36:43 | 401 |     679.477µs |      172.19.0.1 | GET      "/api/channels/1?from=2022-09-20&to=2022-09-27"
[GIN] 2022/09/26 - 16:36:43 | 304 |     123.782µs |      172.19.0.1 | GET      "/favicon-32x32.png"
[GIN] 2022/09/26 - 16:36:43 | 304 |      54.708µs |      172.19.0.1 | GET      "/login"
[GIN] 2022/09/26 - 16:36:48 | 200 |     581.892µs |      172.19.0.1 | POST     "/api/login"
[GIN] 2022/09/26 - 16:36:48 | 200 |    4.343166ms |      172.19.0.1 | GET      "/api/settings"
[GIN] 2022/09/26 - 16:36:48 | 200 |   42.187698ms |      172.19.0.1 | GET      "/api/flow?from=2022-09-20&to=2022-09-27&chan_id=1"
[GIN] 2022/09/26 - 16:36:48 | 200 |  109.047513ms |      172.19.0.1 | GET      "/api/channels/1?from=2022-09-20&to=2022-09-27"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xf81dfe]

goroutine 45 [running]:
github.com/lncapital/torq/pkg/lnd.insertRoutingPolicy(0xc0004ed650, {0xc000702800?, 0x1?, 0x0?}, 0x0, 0xc0006a8c40)
	/app/pkg/lnd/channelgraph.go:178 +0x35e
github.com/lncapital/torq/pkg/lnd.ImportRoutingPolicies({0x1659c60, 0xc0008a82f0}, 0xc0004ed650, {0xc000702800, 0x1, 0x1})
	/app/pkg/lnd/channelinfo.go:131 +0x4cc
github.com/lncapital/torq/cmd/torq/internal/subscribe.Start({0x1644fc8, 0xc0004f7dc0}, 0xc000816e00, 0xc0004ed650, 0x1)
	/app/cmd/torq/internal/subscribe/server.go:86 +0x339
main.main.func1.2.1({0x1, {0xc000042978, 0x11}, {0xc0008c2380, 0x37f, 0x37f}, {0xc0008c40e0, 0xd9, 0xd9}})
	/app/cmd/torq/torq.go:200 +0x195
created by main.main.func1.2
	/app/cmd/torq/torq.go:183 +0x24f
```

Any suggestions? 

(note my node has a ton of data)

**Comment by dannydeezy** [comment]
seems like some channel may have an undefined value in its Routing Policy, triggering this error?

**Comment by ohenrik** [comment]
@maxwedwards Was this issue just solved by v0.8.17? (nil pointer in routing policy)

**Comment by ohenrik** [comment]
This has been fixed in after version 0.8.16 by 4cee697.
